.golangci-suggestions.yaml

```diff
@@ -0,0 +1,285 @@
+# These linters are optional.
+# See .golangci.yaml for checks which are enforced via CI.
+linters:
+  disable-all: true
+  enable:
+    - asasalint
+    - asciicheck
+    - bidichk
+    - bodyclose
+    - containedctx
+    - contextcheck
+    - decorder
+    - depguard
+    - dogsled
+    - durationcheck
+    - exhaustruct
+    - errcheck
+    - errchkjson
+    - errname
+    - errorlint
+    - exhaustive
+    - exportloopref
+    - forcetypeassert
+    - gci
+    - gochecknoglobals
+    - gochecknoinits
+    - goconst
+    - gocritic
+    - godot
+    - godox
+    - goerr113
+    - gofmt
+    - gofumpt
+    - goimports
+    - goprintffuncname
+    - gosec
+    - gosimple
+    - govet
+    - ireturn
+    - importas
+    - ineffassign
+    - loggercheck
+    - makezero
+    - misspell
+    - musttag
+    - nakedret
+    - nestif
+    - nilerr
+    - nilnil
+    - noctx
+    - nolintlint
+    - nosprintfhostport
+    - paralleltest
+    - prealloc
+    - predeclared
+    - promlinter
+    - reassign
+    - revive
+    - rowserrcheck
+    - staticcheck
+    - stylecheck
+    - tagliatelle
+    - thelper
+    - tparallel
+    - typecheck
+    - unconvert
+    - usestdlibvars
+    - unused
+    - wastedassign
+    - whitespace
```

Comment:
AFAIK we agreed that gomnd is not needed since revive add-constant is active.

Same for testpackage. AFAIK we agreed that this is not needed. This linter wants that the tests are in their own package. 

And interfacebloat is in `interfacebloat` is already in the suggestions-config.

Reply:
Sorry, but I don't see interfacebloat.. It's in the config, but not in the list! And with the other two linters you are right.